src/net/sourceforge/kolmafia/persistence/DebugDatabase.java

```diff
@@ -139,7 +144,7 @@ private static boolean decodedNamesEqual(String name1, String name2) {
   private static final String ITEM_HTML = "itemhtml.txt";
 
   private static final String ITEM_DATA = "itemdata.txt";
-  private static final StringArray rawItems = new StringArray();
+  private static final Map<Integer, String> rawItems = new LinkedHashMap<>();
```

Comment:
Why can't this be an ArrayList? Looking at how this is used, we don't iterate over it (although we could). We just get and set by itemId - and ArrayList works fine for that. Or HashMap.